ketchup/options.py

```diff
@@ -35,9 +41,11 @@ def calc_eps_per_day(total_eps, avail_days, eps_per_day_limit=None):
         return 0  # This avoids division by zero.
 
     if eps_per_day_limit is not None:
-        return min(total_eps / avail_days, eps_per_day_limit)
+        result = min(total_eps / avail_days, eps_per_day_limit)
```

Comment:
Rather than returning it from the if statement or the else statement directly as a variable, it is returned at the end of the if statement block, process through the `math` module and the `ceil` function to convert the float integer we receive as a whole number instead.